src/pages/organizers/[organizerId]/ownerships/index.page.tsx

```diff
@@ -1,22 +1,148 @@
+
+  const getOwnershipRequestsQuery = useGetOwnershipRequestsQuery({
+    organizerId: organizerId,
+  });
+
+  const requestsByState: { [key: string]: OwnershipRequest[] } = useMemo(
+    () =>
+      groupBy(
+        // @ts-expect-error
+        getOwnershipRequestsQuery.data?.member,
+        'state',
+      ),
+    [getOwnershipRequestsQuery],
+  );
+
+  const approvedRequests = requestsByState[RequestState.APPROVED] ?? [];
+  const pendingRequests = requestsByState[RequestState.REQUESTED] ?? [];
+
   return (
     <Page>
-      <Page.Title>Hello world</Page.Title>
+      <Page.Title>
+        {t('organizers.ownerships.title', {
+          name: organizer?.name?.[i18n.language],
+        })}
+      </Page.Title>
       <Page.Content>
-        <div>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore eos
-          nisi maiores. Doloribus molestias magnam facilis! Eum rerum ea fugit
-          excepturi doloribus, assumenda quod vitae magni tempore voluptatum
-          adipisci aliquam.
-        </div>
+        <Inline spacing={5}>
+          <Stack spacing={5} flex={3}>
+            <Alert variant={AlertVariants.PRIMARY} fullWidth>
+              {t('organizers.ownerships.info')}
+            </Alert>
+            <Stack spacing={4}>
+              <Title size={3}>{t('organizers.ownerships.owners')}</Title>
+              <OwnershipsTable
+                requests={approvedRequests}
+                renderActions={() => (
+                  <Button variant={ButtonVariants.ICON}>
+                    <Icon name={Icons.TRASH} />
+                  </Button>
+                )}
+              />
+            </Stack>
+            {pendingRequests.length > 0 && (
+              <Stack spacing={4}>
+                <Title size={3}>{t('organizers.ownerships.pending')}</Title>
+                <OwnershipsTable
+                  requests={pendingRequests}
+                  renderActions={(request) => (
+                    <Inline spacing={3}>
+                      <Button
+                        variant={ButtonVariants.SUCCESS}
+                        iconName={Icons.CHECK_CIRCLE}
+                        spacing={3}
+                      >
+                        {t('organizers.ownerships.table.actions.approve')}
+                      </Button>
+                      <Button
+                        variant={ButtonVariants.DANGER}
+                        iconName={Icons.TIMES_CIRCLE}
+                        spacing={3}
+                      >
+                        {t('organizers.ownerships.table.actions.reject')}
+                      </Button>
+                    </Inline>
+                  )}
+                />
+              </Stack>
+            )}
+          </Stack>
+          <Stack spacing={3.5} flex={1}>
+            <Button variant={ButtonVariants.PRIMARY}>
+              {t('organizers.ownerships.actions.add')}
+            </Button>
+            <Button
```

Comment:
You probably want to use a Link with button primary styling here

Reply:
This is not within the scope of this ticket.